layers/+tools/node/local/npm-and-yarn/npm-and-yarn.el

```diff
@@ -0,0 +1,54 @@
+;; -*- lexical-binding: t -*-
+
+;;; npm-and-yarn.el
+
```

Comment:
Please add a standard Melpa package header, otherwise we cannot accept it into emacs.
Do you also intent to upload the package to Melpa? If it is so I would prefer to replace the locale version once it is uploaded.